DEVELOPMENT.md

```diff
@@ -258,7 +258,7 @@ httpClient.SetOptions(clientOptions)
 1. [Global function pointers](#global-function-pointers)
 1. [Test Parallelization](#test-parallelization)
 
-Unit tests are done using basic `golang` means.
+Unit tests are done using basic `golang` means. As the test files are tagged add the corresponding tag to the run command, as for example `go test -run ^TestRunAbapAddonAssemblyKitCheck$ github.com/SAP/jenkins-library/cmd -tags=unit`. In VSCode this can be done by adding the flag `"-tags=unit"` to the list of `"go.testFlags"` in the `settings.json` of the go extension.
```

Comment:
Do we need the cap(^) and $?
```suggestion
Unit tests are done using basic `golang` means. As the test files are tagged add the corresponding tag to the run command, as for example `go test -run TestRunAbapAddonAssemblyKitCheck github.com/SAP/jenkins-library/cmd -tags=unit`. In VSCode this can be done by adding the flag `"-tags=unit"` to the list of `"go.testFlags"` in the `settings.json` of the go extension.
```

Reply:
It is to ensure that only this test and not a test that contains this string is executed (regex start `^` and end `$`). VS Code does this when you click on running a specific test with their UI. It can be left out of course, but personally I would use that. What do you think?

Reply:
maybe with some example, e.g. 
    "go.testTags": "-tags=unit",
    "go.buildFlags": [
        "-tags=unit"
    ],
    "gopls": {
        "build.buildFlags": [ "-tags=unit"]
    },